api/src/pcapi/core/providers/titelive_utils.py

```diff
@@ -0,0 +1,114 @@
+import csv
+import json
+import os.path
+from pathlib import Path
+import typing
+from typing import TypedDict
+
+from black import FileMode
+from black import format_str
+
+from pcapi import settings
+
+
+DIRECTORY = Path(__file__).resolve().parent
+FILENAME = "titelive_gtl.py"
+FILE_PATH = os.path.join(DIRECTORY, FILENAME)
```

Comment:
(détail / style) Pour être un peu plus moderne, on peut utiliser l'opérateur `/` sur un objet de type `Path`, qui fait la même chose que `os.path.join()` : 

```suggestion
FILE_PATH = DIRECTORY / "titelive_gtl.py"
```